RestAPI_Automation/Tests/Tests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Diagnostics;
+using NUnit.Framework;
+using RestAPI_Automation.Requests;
+
+namespace RestAPI_Automation.Tests;
+
+public class Tests
+{
+    // Получить список юзеров, создать рандомный айдишник (нет в списке юзеров), отправить запрос с этим айдишником и проверить ответ – Not Found
+    [Test]
+    public void Task1()
+    {
+        // getting the list of users
+        var content1 = APIHelper.CreateGetRequest<CreatedUsers>(Constants.getUersEndpoint);
+
+        // verifying that there are correct data in the response
+        Assert.AreEqual(2, content1.page);
+        Assert.AreEqual("Michael", content1.Data[0].first_name);
+
+        // creating random id and verifying that it is successfully created
+        var content2 = APIHelper.CreatePostRequest<UserToCreate>(Payloads.payloadTask1);
+        Assert.AreEqual("Mike", content2.name);
+        Assert.AreEqual(280, content2.id);
+
+        // sending request with new id
+        var response = APIHelper.GetRequest($"?id={content2.id}");
+
+        // verifying that we got "Not Found" response
+        Assert.AreEqual("Not Found", response.StatusDescription);
```

Comment:
better take status code, then you can to compare it with HttpStatusCode.NotFound

Reply:
corrected